### PR TITLE
Clean-up warnings during NuGet build

### PR DESCRIPTION
**Pull request by keveleigh**
## Overview

A few warnings popped up recently in the NuGet build:

image

## Changes
- Fixes: https://github.com/microsoft/MixedRealityToolkit-Unity/issues/6517

### CODE REVIEW

File: Assets/MixedRealityToolkit.SDK/Experimental/NonNativeKeyboard/Scripts/SliderInputField.cs [hunk 1]
```diff
@@ -12,8 +12,10 @@ namespace Microsoft.MixedReality.Toolkit.Experimental.UI
     /// </summary>
     public class SliderInputField : TMP_InputField
     {
+#pragma warning disable 0169
```
Comment (Troy-Ferrell) [inline Assets/MixedRealityToolkit.SDK/Experimental/NonNativeKeyboard/Scripts/SliderInputField.cs]: Can you add comment of what 0169 is and why we disable it?

Comment (keveleigh) [reply Assets/MixedRealityToolkit.SDK/Experimental/NonNativeKeyboard/Scripts/SliderInputField.cs]: Turns out it looks like this code wasn't rendering in the inspector anyway.
I instead made a custom inspector with the experimental feature readout

Comment (keveleigh) [reply Assets/MixedRealityToolkit.SDK/Experimental/NonNativeKeyboard/Scripts/SliderInputField.cs]: image

**Pull request merged**